src/matlab2tikz.m

```diff
@@ -2724,7 +2728,8 @@ function assertRegularAxes(handle)
         cl = 'specgraph.errorbarseries';
 
     % quiver plots
-        % TODO: maybe check for existence of `udata` and absence of `ldata`
+    elseif hasProperties(h, {'udata','vdata'}, {'ldata'})
```

Comment:
A sorry, it was UData, VDAta and WData

http://de.mathworks.com/help/matlab/ref/quiverseries-properties.html